sitestats/sitestats-impl/src/java/org/sakaiproject/sitestats/impl/StatsUpdateManagerImpl.java

```diff
@@ -1567,6 +1607,29 @@ private Optional<Instant> doGetSavedBegin(Session session, String siteId, String
 		}
 	}
 
+	private Optional<Integer> doGetOpenSessions(Session session, String siteId, String userId) {
```

Comment:
this seems strange can't we just always return an Integer or is there some other meaning when NULL is returned?